boot/bootutil/include/bootutil/crypto/ecdsa_p256.h

```diff
@@ -205,6 +205,9 @@ static inline int bootutil_ecdsa_p256_verify(bootutil_ecdsa_p256_context *ctx,
     (void)ctx;
     (void)pk_len;
     (void)sig_len;
+	if (check_compression_byte(&pk) < 0)
```

Comment:
cs: align with lines above,
use pattern :
``` C
if (condition) {
  code
}
```

Reply:
Note: sdk-mcuboot checkpath is misconfigured. I opened ci-ticket already.

Reply:
@Vge0rge I men that this PR should follow coding standard visible in the file aside.